pkgs/tools/networking/curlie/default.nix

```diff
@@ -1,20 +1,29 @@
-{ buildGoModule, fetchFromGitHub, lib }:
+{ buildGoModule, fetchFromGitHub, lib, curlie, testVersion }:
 
 buildGoModule rec {
   pname = "curlie";
-  version = "1.6.7";
+  version = "1.6.9";
 
-  src= fetchFromGitHub {
+  src = fetchFromGitHub {
     owner = "rs";
     repo = pname;
     rev = "v${version}";
-    sha256 = "sha256-uWLJWhsqJaLji2JSuVX8Vu929AdozhtAPwsqXdpEt84=";
+    hash = "sha256-3EKxuEpFm+lp2myMfymYYY9boSXGOF2iAdjtGKnjJK0=";
   };
 
   vendorSha256 = "sha256-tYZtnD7RUurhl8yccXlTIvOxybBJITM+it1ollYJ1OI=";
 
   doCheck = false;
 
+  ldflags = [
+    "-X main.version=${version}"
```

Comment:
```suggestion
    "-s" "-w" "-X main.version=${version}"
```

Reply:
Are those defaults that was there before setting the `ldflags`?

Reply:
no